### DISCUSSION TITLE
ارور در هنگام نصب پیش‌نیاز های کتابخانه

### DISCUSSION THREAD

**Discussion by aaq1386** [kianahmadian/python-bale-bot#154] [Q&A] [answered]
با سلام.
در طی تلاش های مکرر برای نصب مجدد python-bale-bot به اروری بر میخوردم 
در ارور به نصب پیشنیاز اشاره می شد، در صورتی که پیشنیاز نصب و اماده استفاده هست
غیر از این نیاز به نصب بخشی از vscode tool ... داشت که انجام شد ولی هنوز ارور ادامه دارد
متن ارور:
```      building 'aiohttp._websocket' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for aiohttp
Failed to build aiohttp
ERROR: ERROR: Failed to build installable wheels for some pyproject.toml based projects (aiohttp)
```
با تشکر

**Comment by kian-ahmadian** [comment, marked as answer]
درود بر شما عزیز
شما احتمالأ در حال نصب نسخه گیتهاب بوده‌اید، و چون تغییرات جدید کتابخانه چند وقتی هست که پیاده‌سازی نشده‌اند این مشکلات پیش آمدند. انشالله به زودی تغییرات اعمال و این مشکلات رفع خواهند شد.
موفق باشید
